compiler/src/iree/compiler/Dialect/Flow/Transforms/FusionUtils.cpp

```diff
@@ -64,10 +59,14 @@ bool areLinalgOpsFusableUsingTileAndFuse(OpOperand &use) {
   // dispatch.
   if (!producer->hasOneUse()) return false;
 
-  // 4. In consumer the result of producer is accessed using identity indexing.
+  // 4. In consumer the result of producer is accessed using identity or
+  // permutation indexing.
   AffineMap consumerIndexingMap = consumer.getTiedIndexingMap(&use);
-  if (!consumerIndexingMap.isIdentity()) return false;
-
+  if (allowTranspose) {
```

Comment:
I dont think we need this. The interchange should be making this still identity.